README.md

```diff
@@ -1,20 +1,46 @@
-# Catone, the OpenCore config.plist censor
+![macOS](https://img.shields.io/badge/Supported_OC_build:-≥0.8.2-white.svg)
 
-The following script was created to censor sensitive fields in OpenCore's config.plist
+# OC Anonymizer – remove sensitive Data from your config.plist
 
-The reason why I created it is that many EFI I find online contain sensitive fields such as SMBIOS data.
-With this script you won't have any prob at all
+## About
+Python Script for removing sensitive Data from OpenCore's `config.plist` before sharing your Config/EFI folder online. It also changes some settings to default, such as: ScanPolicy, Custom Entries, LauncherOptions, etc. See details below. Basically it removes all those settings specific to your system and user preferences which may not be desireble to have on a different machine.
 
-# Usage
+## Features
 
-```bash
-$> python3 oc_anonymizer.py PATH_TO_CONFIG.plist
-```
+Changes the following Settings/Parameters in the **config.plist**:
 
-It'll create `censored_config.plist` with the stripped sensitive data
+- **SMBIOS** data:
+	- Deletes `PlatformInfo/Generic/MLB`
+	- Deletes `PlatformInfo/Generic/ROM`
+	- Deletes `PlatformInfo/Generic/SystemSerialNumber`
+	- Deletes `PlatformInfo/Generic/SystemUUID`
+- **Security Settings**:
+	- `Misc/Security/ApECID` = `0`
+	- `Misc/Security/ScanPolicy` = `0`
+	- `Misc/Security/SecureBootModel` = `Disabled` &rarr; Disables Apple Secure Boot hardware model to avoid issues during Installation. Re-enable in Post-Install so System Updates work when using an SMBIOS of a Mac model with a T2 Security Chip.
+	- `Misc/Security/Vault` = `Optional` 
+- **Other Settings**:
+	- Sets `Misc/Boot/LauncherOption` to `Disabled` &rarr; To avoid changing Boot Menu entries on the target system
+	- Deletes custom entries from `Misc/BlessOverride`
+	- Deletes custom boot loader entries from `Misc/Entries`
+	- Sets `Misc/Debug/Target` to `3` (Default)
+	- `UEFI/APFS`: Changes `MinDate` and `MinVersion` to `-1` to maximize macOS compatibility
 
-# Credits
+## Instructions
+- Install [**Python**](https://www.python.org/) if you haven't already
+- Download [**OC Anonymizer**](https://github.com/5T33Z0/OC-Anonymizer/archive/refs/heads/master.zip) and unpack it
```

Comment:
This line should be ditched as it's kinda obvious to clone `master`. Maybe adding a note like `clone the repo and do X, Y and Z`

Reply:
I need to keep it dumbed down for average hackintosh users. Most of the don't even now how to download from github or what to clone even means. ;)

Reply:
Alright. At this point I'd rather change the link to point at this repo

Reply:
I could change the description to: Click on "Code" > "Download.zip" than it's link independent.